include/utils.php

```diff
@@ -4501,6 +4481,9 @@ function str_split_php4($string, $length = 1)
     return $return;
 }
 
+/*
+ * WONTFIX: Not updating this.. This should be removed !!!
+ */
 if (version_compare(phpversion(), '5.0.0', '<')) {
```

Comment:
I agree. I think your right. We should remove these statements. We don't support anything less than version 5.3. I ran a quick search across the project. I can't see any benefit for it being kept in. Can you please remove these statements?